docs: Add section on delaying browser agent loader (#16064)
newrelic/docs-website: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

Alternatively, and while we STRONGLY recommend the inline script path, you can also use a src script to insert the agent in an `async` or `defer` manner. For caveats and details, continue below.

### Delaying browser monitoring [#delayed-script]

<Callout variant="caution">
  Delaying instantiation causes the agent to miss information until it is fully loaded and initialized. The order of execution of these scripts does lead to some loss in instrumentation data from the start of page views.
</Callout>

For possible performance or other reasons, you may wish to delay the initialization of the browser agent from the start of the page load cycle. You won't be able to use the `newrelic` API until the agent is fully initialized. This delay also creates incomplete data in your browser monitoring UI.

Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.

You can delay the agent initialization in these ways:

* Add a `<script-src>` tag with the `async` attribute.

> * Add a `<script-src>` tag with the `defer` attribute.

Author: probably remove `<`, `>`

Reviewer: Thanks!

### Browser monitoring only [#copy-paste]

> Use this method if you can't use APM for whatever reason, like if your APM agent isn't supported for auto-injection (as is true for Node.js and some Java apps). If you use this method, you'll receive a customized JavaScript snippet at the end of the install process in the New Relic UI, and you add that to the HTML of any pages you want to monitor in your app/site.

Reviewer: ```suggestion
Use this method if you can't use APM for whatever reason, like if your APM agent isn't supported for auto-injection (as is true for Node.js and some Java apps). If you use this method, the UI provides you with a customized JavaScript snippet at the end of the install process that and you can add to the HTML of any pages you want to monitor on your site.
```

The JavaScript snippet should be pasted within the HTML `<head>` tag of views in your app. The head tag typically handles third party scripts, fonts, and other elements that load on all your app pages, so you'll only need to paste it into one place. Paste it as close to the top of the `<head>` as possible, but after any position-sensitive META tags (such as X-UA-Compatible and charset). For more about the snippet placement, see [JavaScript placement requirements](/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement).

> Alternatively and while we STRONGLY recommend the inline script path, you can also use a src script to insert the agent in an `async` or `defer` manner. For caveats and details, continue below.

Reviewer: ```suggestion
Alternatively, and while we STRONGLY recommend the inline script path, you can also use a src script to insert the agent in an `async` or `defer` manner. For caveats and details, continue below.
```

### Delaying browser monitoring [#delayed-script]

<Callout variant="caution">

>   Delaying instantiation will cause the agent to miss information until such time that it is fully loaded and initialized. We assume you understand the order of execution for these scripts and the loss in instrumentation data from the start of page views.

Reviewer: ```suggestion
  Delaying instantiation causes the agent to miss information until it is fully loaded and initialized. The order of execution of these scripts does lead to some loss in instrumentation data from the start of page views.
```

</Callout>

For possible performance or other reasons, you may wish to delay the initialization of the browser agent from the start of the page load cycle. You won't be able to use the `newrelic` API until the agent is fully initialized. This delay also creates incomplete data in your browser monitoring UI.

Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.

You can delay the agent initialization in these ways:

> * Add a `<script-src>` tag with the `async` attribute.
> * Add a `<script-src>` tag with the `defer` attribute.

Reviewer: ```suggestion
* Add a `script-src` tag with the `async` attribute.
* Add a `script-src` tag with the `defer` attribute.
```

> For some features, early page life data will not be captured. For others, it may be extrapolated in best faith. The agent is still expected to function after it has been initialized, whenever that occurs. The tardiness in which the agent will wrap globals or   subscribe to events relative to other libraries or code will affect captured data as well.

Reviewer: ```suggestion
Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.
```

I deleted the first two sentences because they're vague and not specific enough.

> The agent startup can be postponed in the following ways:

Reviewer: ```suggestion
You can delay the agent initialization in these ways:
```

* dynamic import or otherwise chronologically late placement of loader code

> The `src` should point to a valid agent loader version in our CDN. For example, the url that always contain our most recently released Pro+SPA agent, `https://js-agent.newrelic.com/nr-loader-spa-current.min.js`. Keep in mind that the [CSP requirement](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring#csp) for loading external third-party scripts still holds.

Reviewer: ```suggestion
Your page's `src` must point to a valid agent loader version in our CDN. For example, the URL that always contains our current Pro+SPA agent release is `https://js-agent.newrelic.com/nr-loader-spa-current.min.js`. [CSP is required](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring#csp) to load external third-party scripts.
```
